docs/app/Examples/elements/List/Variations/ListExampleHorizontalNumbered.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Image, List } from 'stardust'
+
+const ListExampleHorizontalNumbered = () => (
```

Comment:
How about we rename this `ListExampleHorizontalOrdered` to keep with the prop name?